scripts/build-and-test-package.sh

```diff
@@ -11,6 +11,5 @@ echo "building and testing package: ${1}"
 ${ROOT_DIR}/scripts/build-package.sh ${1}
 
 pushd ${ROOT_DIR}/packages/${1}
-    npm run unit-test
-    npm run integration-test
+    npx jest --maxWorkers 1
```

Comment:
This was for me for debugging, I don't think we actually want to commit it.

Reply:
Ok, I'll revert it. It did, however, fix a persistent local problem I've been having with tests failing with `Rate exceeded` errors.

Reply:
Yeah it is useful for development.  Feel free to add another bash script that uses that flag, or another command in the package.json scripts section.  But that file that you had changed gets used by CI.